### PR TITLE
Menu 2

### PR DESCRIPTION
**Pull request by KristePe**
Have a look at my code 😄

### CODE REVIEW

File: restaurant-menu.html [hunk 1]
```diff
@@ -22,18 +22,165 @@
       </div>
       <!-- Main Section -->
```
Comment (FrontendSi) [inline restaurant-menu.html:19]: Please add media queries for desktop, tab and mobile versions 😃

File: restaurant-menu.html [hunk 2]
```diff
@@ -22,18 +22,165 @@
...
+
+        <div class="dish-card">
+            <div class="dish-card-top">
+                <img src="https://img.freepik.com/premium-photo/pizza-salami-fast-food-pepperoni-sausage-cheese-tomato-sauce-dough-fresh-meal-snack_88242-14641.jpg?w=2000" alt="">
+            </div>
+
+            <div class="dish-card-bottom">
+                <!-- <h5>MARGHERITA</h5> -->
+                
+                    <ul> Pepperoni
```
Comment (FrontendSi) [inline restaurant-menu.html]: Card's image adjustment

**Review by FrontendSi — COMMENTED** [review]
Please update Your code :)

**Comment by KristePe** [comment]
I'll do my best!

**Pull request merged**